indigo_app/templates/document/_annotations.html

```diff
@@ -2,25 +2,30 @@
 {% verbatim %}
 {{#unless permissions.readonly }}
   <div class="float-right">
-    <a href="#" class="dropdown-toggle btn btn-sm" data-toggle="dropdown"></a>
-    <div class="dropdown-menu">
-      {{#unless in_reply_to}}
-        {{#unless task}}
-          {{#if permissions.can_create_task}}
-            <a href="#" class="dropdown-item create-anntn-task">Create task</a>
+
+    {{#if id}}
+      <a href="#" class="dropdown-toggle btn btn-sm" data-toggle="dropdown"></a>
+    
+      <div class="dropdown-menu">
+        {{#unless in_reply_to}}
+          {{#unless task}}
+            {{#if permissions.can_create_task}}
+              <a href="#" class="dropdown-item create-anntn-task">Create task</a>
+            {{/if}}
+          {{/unless}}
+          {{#if permissions.can_change }}
+            <a href="#" class="dropdown-item close-anntn">Resolve</a>
+            <div class="dropdown-divider"></div>
           {{/if}}
         {{/unless}}
+
         {{#if permissions.can_change }}
-          <a href="#" class="dropdown-item close-anntn">Resolve</a>
```

Comment:
It looks like we've lost the Resolve item?

Off the top of my head, I don't think this template needs to change at all?

Reply:
It was not lost, just moved in the snippet. Also, the template needs to change to disable `Edit` and `Delete` options for fake annotations.

Reply:
cool